README.md

```diff
@@ -77,34 +26,43 @@ Usage:
   ddev [command]
 
 Available Commands:
-  config       Create or modify a ddev application config in the current directory
-  describe     Get a detailed description of a running ddev site.
-  exec         Execute a Linux shell command in the webserver container.
-  hostname     Manage your hostfile entries.
-  import-db    Import the database of an existing site to the local dev environment.
-  import-files Import the uploaded files directory of an existing site to the default public upload directory of your application.
-  list         List applications that exist locally
-  logs         Get the logs from your running services.
-  restart      Restart the local development environment for a site.
-  remove       Remove an application's local services.
-  sequelpro    Easily connect local site to sequelpro
-  ssh          SSH to an app container.
-  start        Start the local development environment for a site.
-  stop         Stop an application's local services.
-  version      print ddev version and component versions
+  auth-pantheon Provide a machine token for the global pantheon auth.
+  config        Create or modify a ddev application config in the current directory
+  describe      Get a detailed description of a running ddev site.
+  exec          Execute a shell command in the container for a service. Uses the web service by default.
+  help          Help about any command
+  hostname      Manage your hostfile entries.
+  import-db     Import the database of an existing site to the local dev environment.
+  import-files  Import the uploaded files directory of an existing site to the default public upload directory of your application.
+  list          List applications that exist locally
+  logs          Get the logs from your running services.
+  pull          Import files and database using a configured provider plugin.
+  remove        Remove the local development environment for a site.
+  restart       Restart the local development environment for a site.
+  sequelpro     Easily connect local site to sequelpro
+  ssh           Starts a shell session in the container for a service. Uses web service by default.
+  start         Start the local development environment for a site.
+  stop          Stop the local development environment for a site.
+  version       print ddev version and component versions
+
+Flags:
+  -h, --help          help for ddev
+  -j, --json-output   If true, user-oriented output will be in JSON format.
 
 Use "ddev [command] --help" for more information about a command.
 ```
 
-## Getting Started - Documentation
-Once you've installed ddev, check out the [ddev Documentation Site](https://ddev.readthedocs.io) for information on how to get started and how to use ddev.
-
 ## Support
 If you've encountered trouble using ddev, please use these resources to get help with your issue:
```

Comment:
"If you're having trouble using ddev" is simpler.

Reply:
+1